manifest.konnector

```diff
@@ -27,6 +27,16 @@
   ],
   "screenshots": [],
   "permissions": {
+    "files": {
+      "type": "io.cozy.files",
+      "verbs": [
+        "DELETE",
+        "POST",
+        "PUT",
+        "GET"
+      ],
+      "path": "/Settings/Home"
```

Comment:
Pretty sure this is not the right syntax see my previous comment

Reply:
You can easily test that : 

If you can make a write outside of this path, then this is not good

Reply:
After many tries I finally ask Bruno his advice regarding this type of path permissions. Here is what's comes out of it: 

Files have no path in CouchDB, only directories. What we could have try was is putting the id of the directory but as we don't know it yet when konnector is installed, we cannot put the permissions in the manifest. We need the Home to give the permission to the konnector after creating the directory. So out of the scope of a konnector.

How did I handle it with this new infos ? Forgetting about the path specific permission ?